source/modulo_core/include/modulo_core/concepts.hpp

```diff
@@ -16,4 +33,15 @@ concept CoreT = std::derived_from<T, state_representation::State> || CorePrimiti
 template<typename T>
 concept CustomT = !CoreT<T>;
 
+template<typename T>
+concept TranslatedT = std::same_as<T, geometry_msgs::msg::Accel> || std::same_as<T, geometry_msgs::msg::AccelStamped>
```

Comment:
this concept covers all message types that are supported by modulo translators - I think by definition that includes `std::same_as<T, EncodedState>`. I would also call it `TranslatedMsgT`, `TranslatableMsgT` or `SupportedMsgT` to distinguish from the other concepts which concern "internal" data types rather than "external" message types